web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -1236,57 +1232,77 @@ class AppVersionHistory extends Component {
-                                className="icon preflightChecks--icon u-marginRight--10 u-cursor--pointer"
-                                data-tip="View preflight checks" />
-                              <ReactTooltip effect="solid" className="replicated-tooltip" />
-                            </div>
-                            <div>
-                              <span className="icon deployLogs--icon u-cursor--pointer" onClick={() => this.handleViewLogs(currentDownstreamVersion, currentDownstreamVersion?.status === "failed")} data-tip="View deploy logs" />
-                              <ReactTooltip effect="solid" className="replicated-tooltip" />
-                              {currentDownstreamVersion?.status === "failed" ? <span className="icon version-row-preflight-status-icon preflight-checks-failed-icon logs" /> : null}
-                            </div>
-                            {app.isConfigurable &&
+                              </div>
                               <div>
-                                <Link to={`/app/${app?.slug}/config/${app?.downstream?.currentVersion?.parentSequence}`} className="icon configEdit--icon u-cursor--pointer" data-tip="Edit config" />
+                                <span className="icon deployLogs--icon u-cursor--pointer" onClick={() => this.handleViewLogs(currentDownstreamVersion, currentDownstreamVersion?.status === "failed")} data-tip="View deploy logs" />
                                 <ReactTooltip effect="solid" className="replicated-tooltip" />
-                              </div>}
-                          </div> : null}
+                                {currentDownstreamVersion?.status === "failed" ? <span className="icon version-row-preflight-status-icon preflight-checks-failed-icon logs" /> : null}
+                              </div>
+                              {app.isConfigurable &&
+                                <div>
+                                  <Link to={`/app/${app?.slug}/config/${app?.downstream?.currentVersion?.parentSequence}`} className="icon configEdit--icon u-cursor--pointer" data-tip="Edit config" />
+                                  <ReactTooltip effect="solid" className="replicated-tooltip" />
+                                </div>}
+                            </div> : null}
+                        </div>
                       </div>
                     </div>
                   </div>
                 </div>
-              </div>
+              }
+
 
               <div className={`flex-column flex1 alignSelf--start ${gitopsEnabled ? "gitops-enabled" : ""}`}>
                 <div className={`flex-column flex1 version ${showDiffOverlay ? "u-visibility--hidden" : ""}`}>
-                {versionHistory?.length > 0 ?
-                  <div>
-                    <div className="TableDiff--Wrapper">
+                {(versionHistory.length === 0 && gitopsEnabled) || versionHistory?.length > 0 ?
+                  <>
```

Comment:
would this mean the same as `(versionHistory?.length >= 0 && gitopsEnabled)` 
it's nit picky, however I think might make it look easier to read.